app/models/concerns/package_filtering.rb

```diff
@@ -40,10 +40,12 @@ def filter(options={})
       location = options['location']
```

Comment:
I think we're missing the `dispatched` filter 🤔

Reply:
Dispatched is already implemented in `state_filters` array. On this line
https://github.com/crossroads/api.goodcity/pull/998/files#diff-5cb04308b5cbd66a34bdcc27a04e0759R44